cli/tests/testdata/test/collect/deno.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+  "test": {
+    "files": {
+      "exclude": ["./collect/ignore"]
```

Comment:
One more test like this, but with "include" and this should be good to merge.

Reply:
What if I add the include to this current one?

Reply:
We just need a test that demonstrates they both work. I think there's only two files in this `collect` directory so adding an include might not demonstrate it works.